core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/TissueOrigin.java

```diff
@@ -5,18 +5,12 @@
 
 import com.eaglegenomics.simlims.core.User;
 
-public interface TissueOrigin extends Aliasable, Serializable {
+public interface TissueOrigin extends Aliasable, Deletable, Serializable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)